src/kritiG/Array4.java

```diff
@@ -0,0 +1,43 @@
+package kritiG;
+//Largest and smallest number in an unsorted array
+
+import java.util.Scanner;
+
+public class Array4 {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		// Accepting any integers
+		System.out.println("Enter some numbers ");
+		String num = sc.nextLine();
+		String[] numbers = num.split(" ");
+		int[] data = new int[numbers.length];
+
+		for (int i = 0; i < data.length; i++) {
+			data[i] = Integer.parseInt(numbers[i]);
+		}
+		largest(data);
+
+		sc.close();
+	}
+
+	static void largest(int[] list) {
+		int count = list[0];
```

Comment:
Hi
Please think here 
If you take count = list[0]
Might wrong in case of 1,2,3,8,5